src/App.tsx

```diff
@@ -109,6 +110,18 @@ function BasicLayout(): ReactElement {
   const headerLinks = [...useHeaderAuthLinks()];
   const location = useLocation();
 
+  const auth = useSblAuth();
+  const navigate = useNavigate();
```

Comment:
Unused

Reply:
Good catch @meissadia. I had bigger dreams for error handling here, but for a future #696. (also my after save actions for cleanup stuff are dead still on the new version of VS Code 😔)

Reply:
[Updated](https://github.com/cfpb/sbl-frontend/pull/697/commits/40988b22e9b3b34b485f6b0caba5c166a96ba393) @meissadia!